src/coin_test/analysis/graphs.py

```diff
@@ -0,0 +1,516 @@
+"""Generate plots for analysis."""
```

Comment:
This file got huge -> I wonder about splitting it into multiple at some other point

Reply:
At some point it should probably be a `graphs` folder with individual files for graphs yeah.